Assets/SC_Attackforenemyclassobjects.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_Attackforenemyclassobjects : MonoBehaviour
+{
+    private Transform _target;
+    private bool _isReadyToAttackThePlayer; //var to show if he is ready to attack the player
+    [SerializeField] private float attackRate; //how fast an attack 
```

Comment:
Comments are not really needed, since the name of the variables speaks for itself